src/openklant/components/klantinteracties/api/viewsets/actoren.py

```diff
@@ -65,23 +70,27 @@ class ActorViewSet(viewsets.ModelViewSet):
     @transaction.atomic
     def perform_create(self, serializer):
         super().perform_create(serializer)
-        actor = serializer.instance
+        instance = serializer.instance
+        uuid = str(instance.uuid)
```

Comment:
I think you can revert this separate-variables thing back to passing in `str(actor.uuid)` to the `logger.info` call, since the `uuid` variable is only used once